src/presenters/teaser-presenter.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+const moment = require('moment');
```

Comment:
moment is a huge library, and as n-teaser is sometimes required in client side code we don't really want to add it.

Can you look at https://date-fns.org/ and see if that will give you what you want?